bitex/formatters/binance.py

```diff
@@ -62,4 +66,12 @@ def open_orders(self):
 
     def wallet(self):
         """Return namedtuple with given data."""
-        raise NotImplementedError
+        data = self.json(parse_int=str, parse_float=str)['balances']
+        # balances = {d['asset']: d['free'] for d in data}
+        # nametuple can not first is number, and can not more then 255
+        balances = {}
+        for i in data:
+            available = float(i['free'])
+            if available > 0:
+                balances[i['asset']] = available
+        return super(BinanceFormattedResponse, self).wallet(balances, self.received_at)
```

Comment:
No number conversions please; bitex returns all of its data as strings in order to be as loss-less as possible.

Reply:
All my code modified, return as strings

Reply:
But......I think number is more convenience for use, these loss-less seem not so important. trans to number before use is inconvenience...

Reply:
Perhaps to you - it depends on the use case. Also, it's not much an inconvenience - libraries like `pandas` convert strings to floats on the fly without any additional parameters required.

Finally, the goal of `bitex` is to provide data as is in a convenient matter, and not read the user's mind of whether or not you need a float or string.